File: Python For Everyone, Horstmann/Chapter2/P2.5.py
# Enhance the output of Exercise P2.4 so that the numbers are properly aligned:
# Sum: 45
# Difference: -5
# Product: 500
# Average: 22.50
# Distance: 5
# Maximum: 25
# Minimum: 20

number1 = int(input("Enter number one: "))
number2 = int(input("Enter number two: "))

numbersSum = number1 + number2
numbersDifference = number1 - number2
numbersProduct = number1 * number1
numbersDistance = abs(number1 - number2)
numbersMax = max(number1, number2)
numbersMin = min(number1, number2)


print("The sum is:        %3d" % numbersSum)
print("The difference is: %3d" % numbersDifference)
print("The product is:    %3d " % numbersProduct)
print("The distance is:   %3d" % numbersDistance)
print("The maximum is:    %3d" % numbersMax)
print("The minimum is:    %3d" % numbersMin)

File: Java-for-Everyone-Late-Objects-Horstmann/Chapter-2-Fundamental-Data-Types/P2_5.java
import java.util.Scanner;

//Enhance the output of P2_4.java so that the numbers are properly aligned.


public class P2_5 {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		System.out.print("Number a: ");
		double number_a = input.nextDouble();
		System.out.print("Number b: ");
		double number_b = input.nextDouble();
		input.close();
		double sum = number_a + number_b;
		double difference = number_a - number_b;
		double product = number_a * number_b;
		double average = (number_a + number_b) / 2;
		double distance = Math.abs(difference);
		double maximum = Math.max(number_a, number_b);
		double minimum = Math.min(number_a, number_b);
		System.out.printf("The sum: %10.2f\n"
				+ 		  "The difference: %10.2f\n"
				+         "The product: %10.2f\n"
				+         "The average: %10.2f\n"
				+         "The distance: %10.2f\n"
				+         "The maximum: %10.2f\n"
				+  		  "The minimum: %10.2f", sum, difference, product, average, distance, maximum, minimum);
	}

}
